packages/react/src/Grid/Grid.features.stories.tsx

```diff
@@ -0,0 +1,289 @@
+          <ColumnNumber>4</ColumnNumber>
+        </Grid.Column>
+      </Grid>
+      <Grid {...args}>
+        <Grid.Column span={9}>
+          <ColumnNumber>9</ColumnNumber>
+        </Grid.Column>
+        <Grid.Column span={3}>
+          <ColumnNumber>3</ColumnNumber>
+        </Grid.Column>
+      </Grid>
+      <Grid {...args}>
+        <Grid.Column span={10}>
+          <ColumnNumber>10</ColumnNumber>
+        </Grid.Column>
+        <Grid.Column span={2}>
+          <ColumnNumber>2</ColumnNumber>
+        </Grid.Column>
+      </Grid>
+    </>
+  )
+}
+
+export const Nesting = args => {
+  return (
+    <Grid {...args}>
+      <Grid.Column>
+        <Grid {...args}>
+          <Grid.Column span={{xsmall: 12, medium: 4}}>
+            <Grid {...args}>
+              <Grid.Column span={{xsmall: 12, medium: 6}}></Grid.Column>
+              <Grid.Column span={{xsmall: 12, medium: 6}}></Grid.Column>
+            </Grid>
+          </Grid.Column>
+          <Grid.Column span={{xsmall: 12, medium: 4}}></Grid.Column>
+          <Grid.Column span={{xsmall: 12, medium: 4}}></Grid.Column>
+        </Grid>
+      </Grid.Column>
+    </Grid>
+  )
+}
+
+export const Responsive = args => {
+  return (
+    <Grid {...args}>
+      <Grid.Column span={{xsmall: 12, small: 12, medium: 6, large: 4, xlarge: 3, xxlarge: 2}}></Grid.Column>
+      <Grid.Column span={{xsmall: 6, small: 6, medium: 6, large: 4, xlarge: 3, xxlarge: 2}}></Grid.Column>
+      <Grid.Column span={{xsmall: 6, small: 6, medium: 6, large: 4, xlarge: 3, xxlarge: 2}}></Grid.Column>
+      <Grid.Column span={{xsmall: 12, small: 12, medium: 6, large: 4, xlarge: 3, xxlarge: 2}}></Grid.Column>
+      <Grid.Column span={{xsmall: 6, small: 6, medium: 6, large: 4, xlarge: 3, xxlarge: 2}}></Grid.Column>
+      <Grid.Column span={{xsmall: 6, small: 6, medium: 6, large: 4, xlarge: 3, xxlarge: 2}}></Grid.Column>
+      <Grid.Column span={{xsmall: 12, small: 12, medium: 6, large: 4, xlarge: 3, xxlarge: 2}}></Grid.Column>
+      <Grid.Column span={{xsmall: 6, small: 6, medium: 6, large: 4, xlarge: 3, xxlarge: 2}}></Grid.Column>
+      <Grid.Column span={{xsmall: 6, small: 6, medium: 6, large: 4, xlarge: 3, xxlarge: 2}}></Grid.Column>
+      <Grid.Column span={{xsmall: 12, small: 12, medium: 6, large: 4, xlarge: 3, xxlarge: 2}}></Grid.Column>
+      <Grid.Column span={{xsmall: 6, small: 6, medium: 6, large: 4, xlarge: 3, xxlarge: 2}}></Grid.Column>
+      <Grid.Column span={{xsmall: 6, small: 6, medium: 6, large: 4, xlarge: 3, xxlarge: 2}}></Grid.Column>
+    </Grid>
+  )
+}
+
+export const Centering = args => {
+  return (
+    <Grid {...args}>
+      <Grid.Column span={6} start={4}></Grid.Column>
+    </Grid>
+  )
+}
+
+export const FiftyFifty = args => {
+  return (
+    <Grid {...args}>
+      <Grid.Column span={6}></Grid.Column>
+      <Grid.Column span={6}></Grid.Column>
+    </Grid>
+  )
+}
+FiftyFifty.storyName = 'Ratio: 50:50'
+
+export const SixtyForty = args => {
```

Comment:
I was just curious why this was named SixtyForty? Is it just because it is approximate? Will there be a need for a 60:40 grid that doesn't conform to our 12 column grid?
<img width="735" alt="image" src="https://user-images.githubusercontent.com/31220082/234005958-12451988-9b75-4d0b-8ffa-896f1dea1e86.png">

Reply:
It is approximate, yes. Many of our site designs use this column config, which is referred to as 60:40, but is actually like 58% and 42% in practice.